frontend/src/components/PaperDetailPage.css

```diff
@@ -0,0 +1,19 @@
+.detail-wrapper {
+    width: 1000px;
```

Comment:
what happens if you resize your browser window?

does a relative or percentage make more sense here?